components/basic-analyzers/src/main/java/org/datacleaner/beans/MockAnalyzer.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.beans;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.apache.metamodel.util.LazyRef;
+import org.datacleaner.api.Analyzer;
+import org.datacleaner.api.AnalyzerResult;
+import org.datacleaner.api.ColumnProperty;
+import org.datacleaner.api.Concurrent;
+import org.datacleaner.api.Configured;
+import org.datacleaner.api.Description;
+import org.datacleaner.api.InputColumn;
+import org.datacleaner.api.InputRow;
+import org.datacleaner.result.AnalyzerResultFuture;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Named("MockAnalyzer")
+@Description("For testing purposes. Sleeps for 5 seconds.")
+@Concurrent(true)
+public class MockAnalyzer implements Analyzer<AnalyzerResultFuture<AnalyzerResult>> {
```

Comment:
Actually we don't want this class in src/main/java ... we could accept it in src/test/java but otherwise it's gonna enter the main distro of DC and that doesn't work! :-)

Note: Your main method would then also need to be in src/test/java